reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java

```diff
@@ -40,30 +41,45 @@
    * @param args program arguments.
    */
   public static void main(String... args) {
+    System.out.println(
+        "message size: "
+            + Configurations.MESSAGE_LENGTH
+            + ", number of messages: "
+            + Configurations.NUMBER_OF_MESSAGES
+            + ", address: "
+            + Configurations.MDC_ADDRESS
+            + ", port: "
+            + Configurations.MDC_PORT);
+
+    LoopResources loopResources = LoopResources.create("rsocket-netty");
 
     TcpClient tcpClient =
         TcpClient.create(ConnectionProvider.newConnection())
-            .runOn(LoopResources.create("client", 1, true))
+            .runOn(loopResources)
             .host(Configurations.MDC_ADDRESS)
-            .port(Configurations.MDC_PORT);
+            .port(Configurations.MDC_PORT)
+            .option(ChannelOption.TCP_NODELAY, true)
+            .option(ChannelOption.SO_KEEPALIVE, true)
+            .option(ChannelOption.SO_REUSEADDR, true)
+            .doOnConnected(System.out::println);
 
     RSocket client =
         RSocketFactory.connect()
             .frameDecoder(Frame::retain)
             .transport(() -> TcpClientTransport.create(tcpClient))
             .start()
+            .doOnSuccess(System.out::println)
             .block();
 
     Disposable report = startReport();
 
-    Supplier<Payload> payloadSupplier = () -> ByteBufPayload.create(BUFFER.retainedSlice());
-
-    Flux.range(1, (int) Configurations.NUMBER_OF_MESSAGES)
+    Mono.fromCallable(() -> ByteBufPayload.create(BUFFER.retainedSlice()))
+        .repeat(Configurations.NUMBER_OF_MESSAGES)
```

Comment:
`Mono.fromCallable` + `.repeat` is much slower than `Flux.range(0, (int) Configurations.NUMBER_OF_MESSAGES`

Reply:
Ok
I will revert to   FLux.range

Reply:
Reverted to Flux range